modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/web/ModelController.java

```diff
@@ -46,8 +46,7 @@
     @ApiOperation(value = "Add model hyper params")
     @PostMapping(value = "/addModelParams", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
The REST URIs should generally be resources, not operations. I would expect to see something like

/models/:uuid

and then do an HTTP PUT against that. Are we really _adding_ params here, or just setting them? (The params _are_ the model...)